.github/workflows/js-sdk-release-tool-pull-request-check.yml

```diff
@@ -0,0 +1,41 @@
+name: Javascript SDK Release Tool Pull Request Check
```

Comment:
The github action needs to be reverted and we should continue to use the Devops pipeline ci.yml

Reply:
https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/ci.yml

Reply:
Thanks the guidance, removed